crates/composition/tests/composition/object_fields_clash/supergraph.graphql

```diff
@@ -0,0 +1 @@
+The field `name` on `Band` is defined in two subgraphs (`label` and `critics`).
```

Comment:
We probably want a separate file for errors.

Reply:
This is not graphql, so yes.